### YamlMime:PythonClass
uid: azure.common.AzureConflictHttpError
name: AzureConflictHttpError
fullName: azure.common.AzureConflictHttpError
module: azure.common
inheritances:
- azure.common.AzureHttpError
constructor:
  syntax: AzureConflictHttpError(message, status_code, *args, **kwargs)
  parameters:
  - name: message
    isRequired: true
  - name: status_code
    isRequired: true
methods:
- uid: azure.common.AzureConflictHttpError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.common.AzureConflictHttpError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.common.AzureConflictHttpError.args
  name: args
